packages/dashboard/src/components/Wizard/Wizard.Base.tsx

```diff
@@ -3,13 +3,26 @@ import { IWizardProps, IWizardStyles, IWizardStyleProps } from './Wizard.types';
 import { SubwayNav } from '../SubwayNav/SubwayNav';
 import { classNamesFunction } from 'office-ui-fabric-react/lib/Utilities';
 import { getStepToShow } from './Wizard.utils';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
+import { wizardAnimationDurationMilliSec } from './Wizard.styles';
 
 const getClassNames = classNamesFunction<IWizardStyleProps, IWizardStyles>();
 
+// This returns a childFactory to provide to TransitionGroup
+// tslint:disable:no-any
+const childFactoryCreator = (classNames: any) => (child: any) =>
```

Comment:
Do we actually need this since we already provide the exit animation class? Docs say it's to apply reactive updates on exit but not sure we need that here since the transition component already does that for us.

Reply:
I need this because to solve the issue where when we reverse the direction of slide, the content is retaining the "Exit" class of opposite direction.